basic-javascript/project-2.spec.js

```diff
@@ -132,38 +131,189 @@ describe('isInRange', () => {
   });
 })
 
-// const isInteger = num => {
-//   if (Math.floor(num) === num) {
-//     return true;
-//   }
-//   return false;
-// };
+  // isInteger
+describe('isInteger', () => {
 
-// const fizzBuzz = num => {
-//   if (num % 5 === 0 && num % 3 === 0) {
-//     return 'fizzbuzz';
-//   } else if (num % 5 === 0) {
-//     return 'buzz';
-//   } else if (num % 3 === 0) {
-//     return 'fizz';
-//   }
-//   return num;
-// };
+  it('determines if a parameter is an integer', () => {
+    expect(isInteger(35)).toBe(true);
+    expect(isInteger(Math.PI)).toBe(false);
+  });
 
-// const isPrime = num => {
-//   if (num < 0) {
-//     return false;
-//   }
-//   if (num === 1 || num === 0) {
-//     return false;
-//   }
-//   for (let i = 2; i < num; i++) {
-//     if (num % i === 0) {
-//       return false;
-//     }
-//   }
-//   return true;
-// };
+  test('returns a boolean', () => {
+    const input1 = Math.random()
+    const result = isInteger(input1);
+    expect(typeof result === 'boolean').toBeTruthy();
+  });
+
+  test('returns true if the number is zero or a negative integer', () => {
+        
+    // Arrange: setup the world
+    const input1 = 0;
+    const input2 = -10;
+    const input3 = -5.5
+    // Act: execute the code you're testing
+    const result = isInteger(input1);
+    const result2 = isInteger(input2);
+    const result3 = isInteger(input3);
+    // Assert: check that it works
+    // expect(getBiggest(input1, input1)).toBe(input1);
+    expect(result).toBe(true);
+    expect(result2).toBe(true);
+    expect(result3).toBe(false);
+  });
+
+  test('expects the parameter to be a number, not a string', () => {
+    expect(isInteger('35')).toBeFalsy();
+    expect(isInteger(35)).toBeTruthy();
+    expect(isInteger()).toBeFalsy();
+  });
+})
+
+// fizzBuzz
+describe('fizzBuzz', () => {
+
+  it('determines if a parameter a multiple of 3 and/or 5', () => {
+    expect(fizzBuzz(15)).toBe('fizzbuzz');
+    expect(fizzBuzz(3)).toBe('fizz');
+    expect(fizzBuzz(5)).toBe('buzz');
+    expect(fizzBuzz(Math.PI)).toBe(Math.PI);
+  });
+
+  test('returns a string or a number', () => {
```

Comment:
Good to explore this options, but this will probably be redundant with the checks you're doing above.
I would also not rely on Math.random except as supplementary or brute force testing